Tests/TestingApps/PurchaseTesterSwiftUI/Shared/PurchaseTesterApp.swift

```diff
@@ -11,7 +11,18 @@ import RevenueCat
 
 @main
 struct PurchaseTesterApp: App {
-    @UIApplicationDelegateAdaptor(AppDelegate.self) var appDelegate
+
+    private let delegate = Delegate()
+
+    init() {
+        Purchases.logLevel = .debug
+        Purchases.configure(with: Configuration
+            .builder(withAPIKey: Constants.apiKey)
+            .with(usesStoreKit2IfAvailable: true)
+            .build())
```

Comment:
Nitpick. I know this is how it was before but it was a bit hard to read at the beginning, should we store the configuration in a variable and pass that to the `configure` method? Would improve readability a bit I think.

Reply:
Done.